src/test/java/link/LinkedListTest.java

```diff
@@ -1,19 +1,25 @@
 package link;
+import org.hamcrest.core.IsNull;
 import org.junit.Test;
 
 import java.io.ByteArrayOutputStream;
 import java.io.OutputStream;
 import java.io.PrintStream;
 import java.util.ArrayList;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.core.IsNull.nullValue;

```

Comment:
These look cool, but you don't seem to be using them; try to clean up your imports before you finish your assignments!

Reply:
Thank you for diving into our codes Michelle! I even had an opportunity to chat about it with a couple others who shared the experience, it definitely is giving me the needed motivation to start being more consummate about these things.